onvm/onvm_nflib/onvm_nflib.c

```diff
@@ -741,6 +741,9 @@ onvm_nflib_stop(struct onvm_nf_local_ctx *nf_local_ctx) {
 
         /* Terminate children */
         onvm_nflib_terminate_children(nf_local_ctx->nf);
+        /* Print statistics summary*/
```

Comment:
```suggestion

        /* Print statistics summary */
```

Small nit about comment `*/`. Also thought there should be a line between the call and comment? If not, that's ok too.